libbpf-tools/filetop.bpf.c

```diff
@@ -0,0 +1,110 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+/* Copyright (c) 2021 Hengqi Chen */
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include "filetop.h"
+
+#define MAX_ENTRIES	10240
+
+#define S_IFMT		00170000
```

Comment:
Looks like these defines are coming from `include/uapi/linux/stat.h`. Since they're not making it into `vmlinux.h`, and other tools might want to use these same defs, can they be pulled out into a separate file? Similarly to `libbpf-tools/blk_types.h`

Reply:
Look like the fileslower tool also requires this, let us put them in a separate file.